src/test/scala/uk/gov/homeoffice/rtemailer/database/PostgresDatabaseSpec.scala

```diff
@@ -0,0 +1,83 @@
+package uk.gov.homeoffice.rtemailer.database
+
+import munit.CatsEffectSuite
+import cats.effect._
```

Comment:
Looks like there unused imports here

Reply:
`import munit.CatsEffectSuite` is used here: `class PostgresDatabaseSpec extends CatsEffectSuite`